src/rpc/blockchain.cpp

```diff
@@ -964,6 +964,7 @@ UniValue gettxoutsetinfo(const JSONRPCRequest& request)
 
     CCoinsStats stats;
     FlushStateToDisk();
+    LOCK(cs_main);
```

Comment:
This is incorrect.

Reply:
Oh, thanks. Switching to `UniValue gettxoutsetinfo(const JSONRPCRequest& request) EXCLUSIVE_LOCKS_REQUIRED(cs_main)` instead.